### PR TITLE
feat(std/uuid): add uuid buffer and offset paramter

### PR DESCRIPTION
**Pull request by underfin**

### CODE REVIEW

**Comment by underfin** [comment]
Maybe I met some unrelated error.

##[warning]Failed to download action 'https://api.github.com/repos/actions/checkout/tarball/v1'. Error Response status code does not indicate success: 500 (Internal Server Error).
##[warning]Back off 27.678 seconds before retry.
##[warning]Failed to download action 'https://api.github.com/repos/actions/checkout/tarball/v1'. Error Response status code does not indicate success: 500 (Internal Server Error).
##[warning]Back off 23.256 seconds before retry.
##[error]Response status code does not indicate success: 500 (Internal Server Error).

**Comment by ry** [comment]
Sounds like an unrelated error - you can restart the CI by pushing an empty commit
```
git commit --allow-empty -m bump && git push
```

File: std/uuid/v4.ts [hunk 1]
```diff
@@ -11,11 +11,19 @@ export function validate(id: string): boolean {
   return UUID_RE.test(id);
 }
 
-export function generate(): string {
+export function generate(buf?: number[], offset?: number): string | number[] {
```
Comment (bartlomieju) [inline std/uuid/v4.ts:14]: @underfin please add a test case exercising new parameters

Comment (underfin) [reply std/uuid/v4.ts:14]: Sorry for waste your time...

**Pull request closed without merging**